settings.gradle.kts

```diff
@@ -28,10 +19,11 @@ include(":app")
 include(":network")
 include(":config")
 include(":core")
-include(":featureHome")
-include(":imageLoading")
```

Comment:
We don't need the `imageLoading` module anymore since we migrate to Compose. Also, I think we should name the packages using `snake-case` 🤔

Reply:
for package names I prefer using full lowercase with the rule of thumb that packages should have simple names and at most 2 (shorter) words.
I found that this forces me to find more concise names in order to keep readability of the name as well